backend/tests/expenses/test_expense_service.py

```
@@ -0,0 +1,92 @@
+import pytest
+from unittest.mock import AsyncMock, patch
+from app.expenses.service import ExpenseService
+from app.expenses.schemas import ExpenseCreateRequest, ExpenseSplit, SplitType
+import asyncio
+
+@pytest.mark.asyncio
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)